packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/providers/BasicProvider.test.jsx

```diff
@@ -1,27 +0,0 @@
-/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Moved to: https://github.com/cerner/terra-framework/pull/545/files?short_path=c4a32c9#diff-4d8540ed809ad08ccc51b97b6179a99aR1